src/app/component/errors/mat-snack-bar/mat-snack-bar.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+import { MatSnackBar } from '@angular/material/snack-bar';
+
+@Component({
+  selector: 'app-mat-snack-bar',
+  templateUrl: './mat-snack-bar.component.html',
+  styleUrls: ['./mat-snack-bar.component.scss']
+})
+export class MatSnackBarComponent implements OnInit {
+
+  constructor(public snackBar: MatSnackBar) { }
+
+  ngOnInit() {
+  }
```

Comment:
empty method - in fact, you don't use this hook, so why do you put it here as well as in `import` and `implements`?

Reply:
fixed